pkg/operators/ebpf/converters.go

```diff
@@ -105,10 +106,61 @@ func (i *ebpfInstance) initEnumConverter(gadgetCtx operators.GadgetContext) erro
 	return nil
 }
 
+func (i *ebpfInstance) initStackConverter(gadgetCtx operators.GadgetContext) error {
+	kernelSymbolResolver, err := kallsyms.NewKAllSyms()
+	if err != nil {
+		return err
+	}
+	for _, ds := range gadgetCtx.GetDataSources() {
+		fmt.Println(ds)
+		// TODO: don't know how to assign tag to fields, hardcoding here
+		// for _, in := range ds.GetFieldsWithTag("stack") {
+		for _, in := range []datasource.FieldAccessor{ds.GetField("kernel_stack")} {
```

Comment:
You can use `ds.GetFieldsWithTag("type:gadget_kernel_stack")`, since you already have a typedef for it and type names are automatically applied as tags by the ebpf operator.